cfep-00.md

```diff
@@ -0,0 +1,31 @@
+
+<table>
+<tr><td> Title </td><td> A short title of the proposal </td>
+<tr><td> Status </td><td> Draft | Proposed | Accepted | Rejected | Deferred | Implemented </td></tr>
+<tr><td> Author(s) </td><td> Full Name &lt;full.name@gmail.com&gt;</td></tr>
+<tr><td> Created </td><td> Aug 30, 2016</td></tr>
+<tr><td> Updated </td><td> Aug 30, 2016</td></tr>
+<tr><td> Discussion </td><td> link to the issue where the CFEP is being discussed, NA is circulated initially </td></tr>
+<tr><td> Implementation </td><td> link to the PR for the implementation, NA if not availble </td></tr>
+</table>
+
+## Abstract
+
+A short description of the change being addressed.
+
+## Other sections
+
+Other relevant sections of the proposal.  Common sections include:
+
+    * Specification
+    * Motivation
+    * Rationale
+    * Backwards Compatibility
+    * Alternatives
+    * Sample Implementation
+    * FAQ
+    * Reference
```

Comment:
There is a nice layout below that briefly explains the purpose of each of these sections below. It might be worthwhile to include in the template so people can see it when writing their proposal. Then again it could be noisy. Would like to hear people's thoughts on this generally.